frontend/test/__support__/e2e/cypress-plugins.js

```diff
@@ -67,8 +67,9 @@ module.exports = (on, config) => {
    ********************************************************************/
 
   if (!isQaDatabase) {
-    config.ignoreTestFiles = "**/metabase-db/**";
+    config.ignoreTestFiles = ["**/metabase-db/**", "qa-db.cy.snap.js"];
```

Comment:
Do we still need the `**/metabase-db/**` ignore given that we've deleted the folder?